analyzers/src/SonarAnalyzer.CSharp/Rules/AvoidExcessiveInheritance.cs

```diff
@@ -42,21 +42,21 @@ public class AvoidExcessiveInheritance : ParameterLoadingDiagnosticAnalyzer
         private const string FilteredClassesDefaultValue = "";
         private const int MaximumDepthDefaultValue = 5;
 
-        private static readonly DiagnosticDescriptor rule =
+        private static readonly DiagnosticDescriptor Rule =
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager, false);
-        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(rule);
+        public override ImmutableArray<DiagnosticDescriptor> SupportedDiagnostics { get; } = ImmutableArray.Create(Rule);
 
         [RuleParameter("max", PropertyType.Integer, "Maximum depth of the inheritance tree. (Number)", MaximumDepthDefaultValue)]
         public int MaximumDepth { get; set; } = MaximumDepthDefaultValue;
 
-        private ICollection<Regex> filteredClassesRegex = new List<Regex>();
+        private ICollection<Regex> filters = new List<Regex>();
```

Comment:
On lines 52 and 54 we have 2 fields that are mixed with property definitions. Let's put fields before properties.